src/damage.cpp

```diff
@@ -190,7 +190,11 @@ void ammo_effects( const tripoint &p, const std::set<std::string> &effects )
             g->m.add_field( pt, fd_tear_gas, 3, 0 );
         }
     }
-
+    if( effects.count( "GAS_FUNGICIDAL" ) > 0 ) {
+        for( auto &&pt : g->m.points_in_radius( p, 1, 0 ) ) {
+            g->m.add_field( pt, fd_fungicidal_gas, 4, 0 );
```

Comment:
Good to know. I thought this might let the gas linger longer so it simulates the effect of 'sinking in'. Should I revert it to 3?